e2e/integration/feedbacktests.js

```diff
@@ -1,7 +1,7 @@
 context( 'Test feedback form', () => {
 
     before( () => {
-        cy.visit( Cypress.env( 'QA_HOST' ) );
+        cy.visit( Cypress.env( 'STAGE_HOST' ) );
```

Comment:
This can be reverted back to `QA_HOST`, as all the other tests are running against QA.